sessions/data-quality/making-data-pipelines-in-r.md

```diff
@@ -14,13 +14,13 @@ talk_title_short: Making Data Pipelines in R
 talk_materials_url: ~
 speakers:
 - name: Meghan S Harris
-  affiliation: University at Buffalo
+  affiliation: PCCTC @ Memorial Sloan Kettering
   url:
     webpage: https://www.thetidytrekker.com/
     twitter: https://twitter.com/meghansharris
     github: https://github.com/Meghansaha
     linkedin: https://www.linkedin.com/in/meghan-harris/
-    affiliation: https://medicine.buffalo.edu/departments/family-medicine/research.html
+    affiliation: **https://medicine.buffalo.edu/departments/family-medicine/research.html**
```

Comment:
Do you want to change or remove this link? This one isn't markdown compatible, so it needs to be just the URL. It's fine not to have a link here, too.

Reply:
I'm sorry! Yeah I was just trying to change it as I don't work there anymore. We could remove it altogether.

Reply:
Sounds good, I'll take this URL out. And congrats on the new job! 🎉